core/src/style/textStyleBuilder.h

```diff
@@ -57,7 +57,19 @@ class TextStyleBuilder : public StyleBuilder {
     bool handleBoundaryLabel(const Feature& _feat, const DrawRule& _rule,
                              const TextStyle::Parameters& _params);
 
-    bool checkRule(const DrawRule& _rule) const override { return true; }
+    bool checkRule(const DrawRule& _rule) const override {
```

Comment:
We should move that to implementation file.

Reply:
👍

Reply:
Updated, merging when CI goes green.